packages/reports/src/Components/Errors/error-component.tsx

```diff
@@ -1,23 +1,34 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { PageError, Dialog } from '@deriv/components';
 import { routes } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 
+type TErrorComponent = {
+    header?: string;
+    is_dialog?: boolean;
+    message?: React.ReactElement | string;
```

Comment:
In the proptypes it also had `Object` as an option. Is there a reason you're not adding this as a type?

Reply:
It can not be an object